UPGRADE.md

```diff
@@ -1,3 +1,39 @@
+# Ilios 3 Upgrade Notes
+
+## General steps
+
+1. Back up your database. _Always._
+
+2. Upgrade you codebase via composer.
+
+ ```bash
+cd YOUR_ILIOS_APPLICATION_ROOT 
+sudo -u apache composer install
```

Comment:
Need to tell composer that we are in production with an environmental car here. Don't remember the exact format.